tasks/FindMyMates.tex

```diff
@@ -0,0 +1,52 @@
+\section{Find My Mates [Party Host]}
+
+The robot starts in a room and waits for someone to approach it. The guest knows the name of other guests at the party but doesn’t know how these people look like and where the people are. The robot will gather this information for the guest.
+
+
+\subsection{Main Goal}
+The robot provides the location and unique description of 3 guests of the party as required by another guest.
+
+\noindent\textbf{Reward:} 750 points (250 points per guest).
+
+\subsection{Bonus rewards}
+\begin{enumerate}[nosep]
+	\item Robot providing unique description of a person without having asked such information (75 points)
+\end{enumerate}
+
+
+\subsection{Setup}
+\begin{itemize}
+	\item \textbf{Location}: The robot is in a predefined room at least one person that will approach it.
+
+	\item \textbf{People}: 5 people distributed around the house.
+\end{itemize}
+
+\subsection{Additional rules and remarks}
+\begin{enumerate}[nosep]
+	\item \textbf{Deus ex Machina:} Score reduction for requesting human assistance is applied per guest.
+
+	\item \textbf{Calling the operator for the final answer:} 50\% reduction if the robot does not go back autonomously to the operator to report the information.
+	
+	\item \textbf{Penalty:} If the robot talks to an empty room or does not clearly face a person or crowd when talking (e.g. speaks to a wall), -75 points of penalty.
```

Comment:
Let's remove this

Reply:
Ok.

Reply:
The test should have three more rules in addition to Deus ex Machina,

__Reporting without returning to the operator:__ 50\% reduction in points if reporting the retrieved information without returning to the operator.

This rule is a substitute to providing bonus points for navigating to the operator. If the robot is running out of time, it can tell the information without returning to the operator. This way the robot still gets the points for retrieving the info but with reduction. Also, it can get some points for information gathering if the navigation fails while returning to the operator.
	
__Description of a Person:__ The robot should provide at least three descriptions of a person which may include information about gender, clothing, skin, face, hair, pose, etc.

__Description of Location:__ Not sure how to describe the location if we suppose all the guests are in the same room. It needs to be discussed.

Reply:
This is getting way too complex.

- **Reporting:** Only scored if the robot goes back and delivers the report in time. Otherwise it makes no sense.
- **Description of a person:** No *at least*. If there is a single bald person and the robot is smart enough to say: John is the bald guy in the living room, one descriptor suffices. If, on the other hand, I've 2 tall, Caucasian, bearded, orange-and-black dressed guys, the robot may also need to report that Max is wearing slippers and Loy has baby face.
- **Description location:** Sammy is the pale guy sitting in the coach. Anna is the bearded dude standing next to the lamp. I see no problem there.